server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -1069,14 +1075,15 @@ public final void flush() throws EngineException {
      * Force merges to 1 segment
      */
     public void forceMerge(boolean flush) throws IOException {
```

Comment:
Since you're in on it, would you mind removing this `forceMerge` method.

Reply:
Killed it :)